tracking/tracking.go

```diff
@@ -0,0 +1,40 @@
+package tracking
+
+import (
+	"os"
+	"time"
+
+	log "github.com/Sirupsen/logrus"
+	"github.com/google/uuid"
+	rancher "github.com/rancher/go-rancher/v2"
+)
+
+const UUIDSetting = "install.uuid"
+
+var RancherUUID = loadRancherUUID()
+
+func loadRancherUUID() string {
+	client, err := rancher.NewRancherClient(&rancher.ClientOpts{
+		Url:       os.Getenv("CATTLE_URL"),
+		AccessKey: os.Getenv("CATTLE_ACCESS_KEY"),
+		SecretKey: os.Getenv("CATTLE_SECRET_KEY"),
+		Timeout:   5 * time.Second,
+	})
+	if err != nil {
+		log.Warn(err)
+		return uuid.Nil.String()
+	}
+
+	setting, err2 := client.Setting.ById(UUIDSetting)
```

Comment:
`err` was previously declared, so `:=` can not be used. Would you prefer I define the `setting` var above and use `=`?